Libraries/LibWeb/HTML/WorkerLocation.cpp

```diff
@@ -60,8 +60,6 @@ WebIDL::ExceptionOr<String> WorkerLocation::host() const
 // https://html.spec.whatwg.org/multipage/workers.html#dom-workerlocation-hostname
 WebIDL::ExceptionOr<String> WorkerLocation::hostname() const
```

Comment:
We can remove the ExceptionOr now :^)

Reply:
Done! I very nearly got caught in a deep pile of yaks with this. 😆 Managed to make it out alive after only a couple of changes.